node/src/node/Main.hs

```diff
@@ -75,3 +75,4 @@ main = do
     CLI.printFlags
     let conf = CLI.configurationOptions (CLI.commonArgs commonNodeArgs)
     runProduction $ withConfigurations conf (action args)
+
```

Comment:
:cho:

Reply:
niche

fucking stack doesn't osil to re-link our fucking binaries without changing them

Reply:
Clean 🤷‍♂️

Reply:
I didn't want to trigger CI because of it, so thx for approve

Reply:
But CI failed anyway 🤷‍♂️